### PR TITLE
Line height for titles

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
Capture d’écran 2021-11-05 à 15 49 17

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1476

**Review by AlexiaAlarza — APPROVED** [review]

**Review by AlexiaAlarza — DISMISSED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1476

File: packages/scss/src/theming/_sizes.scss [hunk 1]
```diff
@@ -1,31 +1,40 @@
 $sizes: (
+  // deprecated
```
Comment (BertrandPodevin) [inline packages/scss/src/theming/_sizes.scss:2]: deprecated here ? Why ?

**Comment by BertrandPodevin** [comment]
While not technically breaking change, this PR might easily break some alignments. If we ceate a new tag or notation for `visual breaking change`, this one is a candidate

File: packages/scss/src/theming/_sizes.scss (hunk 1 above)
Comment (vvalentin-lucca) [reply packages/scss/src/theming/_sizes.scss:2]: A design team request.

**Review by AlexiaAlarza — APPROVED** [review]

**Pull request merged**